### ISSUE TITLE
Length Inaccurate on Multi-Channel Audio Files

### ISSUE THREAD

**Issue by kevinbird15** [pytorch/audio#236]
## 🐛 Bug

When doing `torchaudio.info()` on a multi channel audio file, the length is multiplied by however many channels there are.

## To Reproduce

1. `torchaudio.load(dual_channel_path)` Look at the shape
1. `torchaudio.info(dual_channel_path)` Look at the length
1. It will be length of a single channel * # of channels

## Expected behavior

I would expect the length to be the length of one channel, not the length of all of them put together. 

## Environment

 - What commands did you used to install torchaudio (conda/pip/build from source)? Pip
 - If you are building from source, which commit is it?
 - What does `torchaudio.__version__` print? (If applicable) 0.3.0

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

 - PyTorch Version (e.g., 1.0):
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

## Additional context

I think the best way to fix this issue is just to take length and divide it by # of channels.

**Comment by jamarshon** [comment]
Left a comment on the PR 😄 
https://github.com/pytorch/audio/pull/237#issuecomment-521653116

**Comment by mthrok** [comment]
This has been resolved via #726.
